src/test/java/com/bakdata/common_kafka_streams/util/TopicClientTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 bakdata
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.bakdata.common_kafka_streams.util;
+
+import static java.util.Collections.emptyMap;
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import net.mguenther.kafka.junit.EmbeddedKafkaCluster;
+import net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig;
+import net.mguenther.kafka.junit.EmbeddedKafkaConfig;
+import net.mguenther.kafka.junit.TopicConfig;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TopicClientTest {
+
+    private static final int TIMEOUT_SECONDS = 20;
+    private static final Duration CLIENT_TIMEOUT = Duration.ofSeconds(10L);
+    private EmbeddedKafkaCluster kafkaCluster;
+
+    @BeforeEach
+    void setup() throws InterruptedException {
+        final EmbeddedKafkaConfig kafkaConfig = EmbeddedKafkaConfig.create()
+                .withNumberOfBrokers(2)
+                .build();
+        final EmbeddedKafkaClusterConfig clusterConfig = EmbeddedKafkaClusterConfig.create()
+                .provisionWith(kafkaConfig)
+                .build();
+        this.kafkaCluster = provisionWith(clusterConfig);
+        this.kafkaCluster.start();
+        Thread.sleep(TimeUnit.SECONDS.toMillis(TIMEOUT_SECONDS));
```

Comment:
It should work without the timeout

Reply:
I'm unsure about this because I had to restart the build once...